master/usage/configuration/mtu.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Configuring MTU
+---
+
+Depending on the environment Calico is being deployed into it may be
+helpful or even necessary to configure the MTU of the TAP (or veth) that is
+attached to each workload and the tunnel devices if IP-in-IP is enabled.
+
+### Selecting MTU size
+
+Typically the MTU for your workload interfaces should match the network MTU.
+If you need IP-in-IP then the MTU size for both the workload **and** tunnel
+interfaces should be 20 bytes less than the network MTU for your network.
+This is due to the extra 20 byte header that the tunnel will add to each
+packet.
+
+#### Common MTU sizes
+
+| Network MTU | Calico MTU | Calico MTU with IP-in-IP |
```

Comment:
I don't understand what to add as far as VXLAN. Is it adding another column to the MTU table for if the traffic will be routed over VXLAN? And then setting that is just setting it in the CNI config (so it would not require an additional how to section)?